https://redhat.atlassian.net/browse/ACM-34544 updates for grc (#8922)
open-cluster-management/rhacm-docs: governance/e8_scan_policy.adoc

= E8 scan policy

[role="_abstract"]

> The Essential 8 (E8) scan policy references the E8 security profiles used to evaluate master and worker nodes. You must install the Compliance Operator to apply the E8 scan policy.

Author: ```suggestion
The Essential 8 (E8) scan policy contain references to the E8 security profiles for evaluating master and worker nodes. You must install the Compliance Operator to apply the E8 scan policy.
```

> The Essential 8 (E8) scan policy contain references to the E8 security profiles for evaluating master and worker nodes. You must install the Compliance Operator to apply the E8 scan policy.

Author: ```suggestion
The Essential 8 (E8) scan policy contains references to the E8 security profiles for evaluating master and worker nodes. You must install the Compliance Operator to apply the E8 scan policy.
```

The E8 scan policy is a Kubernetes configuration policy in {acm-short}. {ocp-short} supports the E8 scan policy.

.Additional resources

> * xref:../governance/create_policy.adoc#managing-security-policies[Managing security policies] for more information.

Reviewer: ```suggestion
* xref:../governance/create_policy.adoc#managing-security-policies[Managing security policies]
```

Reviewer: <img width="525" height="93" alt="Image" src="https://github.com/user-attachments/assets/bf177e31-f1d3-48eb-9096-4500e5ca6f4a" />

This file is good apart from this line

Author: great catch!Thank you
